webapp/src/main/java/com/box/l10n/mojito/service/tm/search/TextUnitDTONativeObjectMapper.java

```diff
@@ -40,15 +40,21 @@ public TextUnitDTO mapObject(CriteriaResult cr) {
     // implementation to understand why getBoolean doesn't work. This
     // seems to work fine, use this code for now.
     t.setIncludedInLocalizedFile(Boolean.valueOf(includedInLocalizedFile));
-    t.setCreatedDate(new DateTime(cr.getDate(idx++)));
+    Date createdDate = cr.getDate(idx++);
```

Comment:
definitely, can we have a util function for `new DateTime` replacement? the old constructor accepted `null`?